nss-pam-ldapd: include Titanium Cloud changes

New nss-pam-ldapd uses default ldap group ID, so we set
gid in nslcd.conf to ldap.

diff --git a/SPECS/nss-pam-ldapd.spec b/SPECS/nss-pam-ldapd.spec
index 72aa77f..20fe844 100644
--- a/SPECS/nss-pam-ldapd.spec
+++ b/SPECS/nss-pam-ldapd.spec
@@ -49,6 +49,7 @@ Source1:	http://arthurdejong.org/nss-pam-ldapd/nss-pam-ldapd-%{version}.tar.gz.s
 Source2:	nslcd.init
 Source3:	nslcd.tmpfiles
 Source4:	nslcd.service
+Source5:	nslcd.conf
 Patch1:		nss-pam-ldapd-0.8.12-validname.patch
 Patch2:         nss-pam-ldapd-0.8.12-In-nslcd-log-EPIPE-only-on-debug-level.patch
 Patch3:		nss-pam-ldapd-0.8.12-uid-overflow.patch
@@ -166,6 +167,9 @@ mkdir -p -m 0755 $RPM_BUILD_ROOT/etc/tmpfiles.d
 install -p -m 0644 %{SOURCE3} $RPM_BUILD_ROOT/etc/tmpfiles.d/%{name}.conf
 %endif
 
+# WRS
+install -m 644 %{SOURCE5} $RPM_BUILD_ROOT/%{_sysconfdir}/nslcd.conf
+
 %clean
 rm -rf $RPM_BUILD_ROOT
 
@@ -249,12 +253,13 @@ if test "$1" -eq "1" && ! grep -q -F "# $comment" $target 2> /dev/null ; then
 		grep -E '^host[[:blank:]]' $source |\
 		sed -r -e "s,^host[[:blank:]](.*),uri ${scheme}://\1/,g" >> $target
 	fi
+	# WRS: we don't want to change our custom base in nslcd.conf
 	# Base doesn't require any special logic.
-	if grep -E -q '^base[[:blank:]]' $source 2> /dev/null ; then
+	# if grep -E -q '^base[[:blank:]]' $source 2> /dev/null ; then
 		# Comment out the packaged default base and replace it.
-		sed -i -r -e 's,^(base[[:blank:]].*),# \1,g' $target
-		grep -E '^base[[:blank:]]' $source >> $target
-	fi
+		# sed -i -r -e 's,^(base[[:blank:]].*),# \1,g' $target
+		# grep -E '^base[[:blank:]]' $source >> $target
+	# fi
 	# Pull in these settings, if they're set, directly.
 	grep -E '^(binddn|bindpw|port|scope|ssl|pagesize)[[:blank:]]' $source 2> /dev/null >> $target
 	grep -E '^(tls_)' $source 2> /dev/null >> $target
